.prettierrc

```diff
@@ -5,4 +5,4 @@
   "printWidth": 80,
   "singleQuote": true,
   "trailingComma": "all"
-}
\ No newline at end of file
+}
```

Comment:
Did you know that the project is currently not setup to auto-format markdown (and all the code examples in the markdown). It would work if you remove `"parser": "flow",` from this file though :)